content/applications/general/voip/voip_widget.rst

```diff
@@ -0,0 +1,76 @@
+===========
+VoIP widget
+===========
+
+Phone calls
+===========
+
+Make phone calls by clicking on the :guilabel:`📞 (phone)` icon in the top navigation bar. A widget
+appears in the lower left corner of the Odoo dashboard. Freely navigate throughout the database
+while making and receiving calls from the widget.
+
+When receiving call in Odoo, the :abbr:`VoIP (Voice over Internet Protocol)` widget rings and
+displays a notification. To minimize the widget click the :guilabel:`X` icon in the upper right of
+the screen.
+
+.. note::
+   The :abbr:`VoIP (Voice over Internet Protocol)` number is the one provided by Axivox. It can be
+   accessed by navigating to `https://manage.axivox.com/ <https://manage.axivox.com/>`_. After
+   logging into the portal go to :menuselection:`Users --> Outgoing number` (column).
+
+.. image:: voip_widget/call.png
+   :align: center
+   :alt: VoIP call in Odoo.
+
+Troubleshooting
+===============
+
+.. tip::
+   If a *Missing Parameter* message appears in the **Odoo softphone**, refresh the Odoo window and
+   try again.
+
+   .. image:: voip_widget/missing-parameter.png
+      :align: center
+      :alt: "Missing Parameter" error message in the Odoo softphone.
+
+.. tip::
+   If an *Incorrect Number* message appears in the Odoo softphone, make sure to use the
+   international format, leading with the plus (+) sign followed by the international country code.
+   E.g., +16506913277 (where +1 is the international prefix for the United States).
+
+   .. image:: voip_widget/incorrect-number.png
+      :align: center
+      :alt: "Incorrect Number" error message in the Odoo softphone.
+
+Tabs
+====
+
```

Comment:
brief sentence introducing that 3 tabs are present when the VoIP phone widget is opened could be beneficial here.

Reply:
Added this in.